tools/lint/lint.py

```diff
@@ -50,7 +51,8 @@
     CheckFeatures('features.txt'),
     CheckHarnessFeatures(),
     CheckLicense(),
-    CheckNegative()
+    CheckNegative(),
+    CheckFileName()
```

Comment:
This list was meant to be alphabetical. Mind moving this entry (and fixing the location of `CheckFeatures` while you're at it)?